package.json

```diff
@@ -30,21 +30,21 @@
     "postversion": "git push && git push --tags"
   },
   "dependencies": {
-    "lodash": "^4.17.11"
+    "bootstrap": "^3.3.6",
```

Comment:
Would it cause any trouble using latest version ? `4.3.x` if I'm correct

Reply:
The host applications are not yet fully migrated to the latest version of `bootstrap` so I think we can keep this version for now.

The migration should be done once and all packages should follow the bump as well.